modules/apps/product-navigation/product-navigation-site-administration/src/main/resources/META-INF/resources/sites/site_administration_body.jsp

```diff
@@ -129,6 +114,66 @@ Group group = siteAdministrationPanelCategoryDisplayContext.getGroup();
 	</c:if>
 </c:if>
 
+<c:if test="<%= !group.isDepot() && !group.isCompany() %>">
+
+	<%
+	PortletURL portletURL = PortletURLFactoryUtil.create(request, ProductNavigationProductMenuPortletKeys.PRODUCT_NAVIGATION_PRODUCT_MENU, RenderRequest.RENDER_PHASE);
+
+	portletURL.setParameter("mvcPath", "/portlet/pages_tree.jsp");
+	portletURL.setParameter("redirect", themeDisplay.getURLCurrent());
+	portletURL.setParameter("selPpid", portletDisplay.getId());
+	portletURL.setWindowState(LiferayWindowState.EXCLUSIVE);
+	%>
+
+	<aui:script sandbox="<%= true %>">
+		Liferay.componentReady('<portlet:namespace />pagesTreeSidenavToggleId').then(
```

Comment:
This means we're introducing a breaking change, where code that before was working now does not. We don't know how many more instances like this we have spread around DXP. 

Let's focus on fixing the origin rather than the usages.

Reply:
Got it.
So does that means not "hydrating" if we don't `additionalProps` and `propsTransfomer`?

Reply:
What does hydrating means? Why and when would we want to hydrate a component? Is this the case?

Try to think through what's happening and ask yourselves if this is something we want happening, then write down those thoughts with a conclusion as to how to proceed ;)

Reply:
> What does hydrating means? 

As far as I understand it means rendering over the initial content rendered by the server

> Why and when would we want to hydrate a component?

Probably when you want to do change what was rendered by the server 

> Is this the case?

No